bitwarden_license/bit-web/src/app/secrets-manager/projects/project-contents/project-contents.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from "@angular/core";
+import { ActivatedRoute } from "@angular/router";
+import { Observable, switchMap } from "rxjs";
+
+import { ProjectView } from "../../models/view/project.view";
+import { ProjectService } from "../project.service";
+
+@Component({
+  selector: "sm-project-contents",
+  templateUrl: "./project-contents.component.html",
+})
+export class ProjectContentsComponent implements OnInit {
```

Comment:
I think calling this `Project` makes more sense than project content. We're browsing a project.

```suggestion
export class ProjectComponent implements OnInit {
```

Reply:
Just updated it. I did intentionally name it `ProjectContentsComponent` because I wanted someone to easily discern the differences between `project`, `projects`, and `projects-list`. (Plus, all of those are under the parent folder `projects`...) Project is the best name though, so once someone reads through the code they should be able to clearly understand the differences 🙂

Reply:
Yea ... I think we should ensure we stick to a single contention which should hopefully make it clear that plural typically denotes listing entities, and singular is viewing an entity.